tests/rbac/common/addresser/user_test.py

```diff
@@ -36,6 +36,53 @@ def test_address(self):
             addresser.address_is(user_address), addresser.AddressSpace.USER
         )
 
+    def test_unique_id(self):
```

Comment:
doc strings on tests for what they are testing

Reply:
resolved